tests/scripts/framework/run_test.sh

```diff
@@ -268,10 +257,11 @@ find_tests()
 mkdir -p $REPORT_DIR
 TESTS_TO_RUN=$(find_tests ${TEST_LIST[@]})
 
-if [[ "$DOWNLOAD_MODE" == "on" ]]; then
+if [[ "$DOWNLOAD_MODEL" = "on" ]]; then
```

Comment:
AFAIK, you should use the `==` operator with the `[[` command for pattern matching.